<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GerenciadorServicos</class>
 <widget class="QMainWindow" name="GerenciadorServicos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>621</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>621</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestor Apihub 25.03.2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblStatusServico">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Status:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInstalar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;v&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: rgb(255, 255, 255);
    background-color: rgb(140, 140, 140);
    font-weight: bold;
   	font-size:80px;
    border-radius: 5px;
    border: none;
    padding: 10px;
 }


QPushButton:pressed {
    /*background-color: rgb(0, 0,166);  /*mais escuro ao pressionar */
	background-color: rgb(0, 197, 0);
    padding-top: 12px;  /* simula o bot√£o afundando */
}
QPushButton:hover{
    /*background-color: rgb(0, 0,166);  /*mais escuro ao pressionar */
	background-color: rgb(0, 197, 0);
	padding-top: 12px;  /* simula o bot√£o afundando */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnServico">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: rgb(255, 255, 255);
    background-color:  rgb(140, 140, 140);    
	font-weight: bold;
   	font-size:80px;
    border-radius: 5px;
    border: none;
    padding: 10px;
 }


QPushButton:pressed {
    background-color: rgb(217, 83,79); /* mais escuro ao pressionar */
    padding-top: 12px;  /* simula o bot√£o afundando */
}

QPushButton:hover {
    background-color: rgb(217, 83,79); /* mais escuro ao pressionar */
    padding-top: 12px;  /* simula o bot√£o afundando */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEditarEnv">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(0, 0, 0); /* preto */
    color: rgb(255, 255, 255);        /* branco */
    font-weight: bold;
    border-radius: 5px;
    border: none;
    padding: 10px 20px;
    cursor: pointer;
    transition: all 0.3s ease; /* transi√ß√£o suave */
}

QPushButton:hover {
    background-color: rgb(0, 50, 100); /* azul escuro */
    transform: scale(1.05);             /* leve zoom */
    box-shadow: 
        0 0 15px rgba(255, 255, 255, 0.2),  /* brilho sutil */
        0 4px 10px rgba(0, 0, 0, 0.5); /* profundidade */
}
</string>
    </property>
    <property name="text">
     <string>üõ†Ô∏è   ENV APIHUB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAbrirLog">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(0, 0, 0); /* preto */
    color: rgb(255, 255, 255);        /* branco */
    font-weight: bold;
    border-radius: 5px;
    border: none;
    padding: 10px 20px;
    cursor: pointer;
    transition: all 0.3s ease; /* transi√ß√£o suave */
}

QPushButton:hover {
    background-color: rgb(0, 50, 100); /* azul escuro */
    transform: scale(1.05);             /* leve zoom */
    box-shadow: 
        0 0 15px rgba(255, 255, 255, 0.2),  /* brilho sutil */
        0 4px 10px rgba(0, 0, 0, 0.5); /* profundidade */
}
</string>
    </property>
    <property name="text">
     <string>üóíÔ∏è  LOG APIHUB</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatusServico_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Vers√£o:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatusServico_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>25.12.01</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatusServico_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Status: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAbrirDash">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(0, 0, 0); /* preto */
    color: rgb(255, 255, 255);        /* branco */
    font-weight: bold;
    border-radius: 5px;
    border: none;
    padding: 10px 20px;
    cursor: pointer;
    transition: all 0.3s ease; /* transi√ß√£o suave */
}

QPushButton:hover {
    background-color: rgb(0, 50, 100); /* azul escuro */
    transform: scale(1.05);             /* leve zoom */
    box-shadow: 
        0 0 15px rgba(255, 255, 255, 0.2),  /* brilho sutil */
        0 4px 10px rgba(0, 0, 0, 0.5); /* profundidade */
}
</string>
    </property>
    <property name="text">
     <string> DASHBOARD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPainel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(0, 0, 0); /* preto */
    color: rgb(255, 255, 255);        /* branco */
    font-weight: bold;
    border-radius: 5px;
    border: none;
    padding: 10px 20px;
    cursor: pointer;
    transition: all 0.3s ease; /* transi√ß√£o suave */
}

QPushButton:hover {
    background-color: rgb(0, 50, 100); /* azul escuro */
    transform: scale(1.05);             /* leve zoom */
    box-shadow: 
        0 0 15px rgba(255, 255, 255, 0.2),  /* brilho sutil */
        0 4px 10px rgba(0, 0, 0, 0.5); /* profundidade */
}
</string>
    </property>
    <property name="text">
     <string>üõ†Ô∏è   ENV PAINEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogAll">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(0, 0, 0); /* preto */
    color: rgb(255, 255, 255);        /* branco */
    font-weight: bold;
    border-radius: 5px;
    border: none;
    padding: 10px 20px;
    cursor: pointer;
    transition: all 0.3s ease; /* transi√ß√£o suave */
}

QPushButton:hover {
    background-color: rgb(0, 50, 100); /* azul escuro */
    transform: scale(1.05);             /* leve zoom */
    box-shadow: 
        0 0 15px rgba(255, 255, 255, 0.2),  /* brilho sutil */
        0 4px 10px rgba(0, 0, 0, 0.5); /* profundidade */
}
</string>
    </property>
    <property name="text">
     <string> LOG PAINEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDesinstalar">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #C82333;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 10px 20px;
    font-weight: bold;
}

/* Hover com leve brilho e suaviza√ß√£o */
QPushButton:hover {
    background-color: #e02b3c;
    border: 1px solid #ff4a55;      /* Realce */
}
/* Efeito de clique */
QPushButton:pressed {
    background-color: #a91e2c;      /* Tom mais escuro */
    padding-top: 12px;
    padding-bottom: 8px;
    border: 1px solid #7f131e;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAtualizarApi">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
            background-color: rgb(0, 50, 100); /* Azul escuro */
            color: rgb(255, 255, 255);        /* branco */
            font-weight: bold;
            border-radius: 5px;
            border: none;
            padding: 8px 18px;
            transition: all 0.3s ease;
        }

        QPushButton:hover {
            background-color: rgb(0, 80, 150);
            box-shadow: 0 4px 8px rgba(0, 0, 0, 0.3);
        }
        QPushButton:pressed {
            background-color: rgb(0, 30, 80);
            padding-top: 12px;
            padding-bottom: 8px;
        }
    </string>
    </property>
    <property name="text">
     <string>üîÑÔ∏è ATUALIZAR APIHUB</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
